Sources/TreeSitterClient/TreeSitterClient.swift

```diff
@@ -7,7 +7,7 @@ public enum TreeSitterClientError: Error {
     case stateInvalid
     case staleContent
     case queryFailed(Error)
-    case asyncronousExecutionRequired
+    case asynchronousExecutionRequired
```

Comment:
This is **technically** API change, but I think we can just let it slide since I doubt it will be too disruptive.